### DISCUSSION TITLE
Can anyone tell me where is the URL for the latest dependency `iced_native`?

### DISCUSSION THREAD

**Discussion by Amazingkenneth** [hecrj/iced#1916] [Q&A] [answered]
There was a directory named `native` in https://github.com/iced-rs/iced/tree/78924fa2998e2d158132dd5d06b2b8eea1edbc4b , but then in https://github.com/iced-rs/iced/tree/435b54e57ecdaf08b1f36626f0000438895c909d it is gone.

I wanted to use the latest source code of `iced_native` but encountered the following error:
```
error: no matching package named `iced_native` found
location searched: https://github.com/iced-rs/iced.git
required by package `graduate v0.1.0 (/home/runner/work/graduate/graduate)`
```

**Comment by LTeder** [comment, marked as answer]
`iced_native` was renamed to `iced_runtime` in 99e0a71504456976ba88040f5d1d3bbc347694ea as part of #1748.

**Comment by Amazingkenneth** [comment]
Thanks!
